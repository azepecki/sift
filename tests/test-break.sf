def int main() {
    arr<str> names = ["plt","os","esaas"];
    for(int i=0; i< len(names); i = i + 1){    
        if (names[i] == "os"){
            break;
        }
        printf(names[i]);
    }
    return 0;
}