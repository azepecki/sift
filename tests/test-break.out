"plt"
"os"
"esaas"
0